packages/@office-iss/react-native-win32/src-win/Libraries/Components/View/View.win32.js

```diff
@@ -253,9 +245,6 @@ const View: React.AbstractComponent<
                   : importantForAccessibility
               }
               nativeID={id ?? nativeID}
-              style={style}
```

Comment:
So we just don't set style anymore?

Reply:
it's still being set through otherProps, they just realized that the flattening was not necessary.

https://github.com/facebook/react-native/pull/44352